test/modules/dndImport/components/DndImportPanel.test.js

```diff
@@ -499,9 +499,11 @@ describe('DndImportPanel', () => {
 
 				simulateDragDropEvent('drop', dataTransferMock, dropZone);
 
-				expect(store.getState().notifications.latest.payload.content).toBe('dndImport_import_unsupported');
```

Comment:
All spies upon SourceTypeService#forBlob have to emulate its now async behavior
e.g. `spyOn(sourceTypeService, 'forBlob').and.resolveTo(sourceTypeResultMock)`